iroha-cli/main.cpp

```diff
@@ -115,6 +124,14 @@ int main(int argc, char* argv[]) {
     std::ofstream output_file("genesis.block");
     output_file << jsonToString(doc);
     logger->info("File saved to genesis.block");
+  } else if (FLAGS_interactive) {
+    // TODO: add login logic (e.g. password check)
```

Comment:
It is actually somehow related, to work with any transaction or query we need account name, it's private and public for signatures. I think it is better to leave comment so that in future we will not forget about it.

Reply:
Okey